stdlib/format.mli

```diff
@@ -45,7 +45,8 @@
 
    {b Warning}: Since {{!section:formatter}formatters} contain
    mutable state, it is not thread-safe to use the same formatter on multiple
-   domains in parallel without synchronization.
+   domains in parallel without synchronization. This may result in
+   [Invalid_argument] to be raised or an undefined behavior.
```

Comment:
"undefined behavior" is understood as "this can segfault", so it is not the correct wording here. I would use: "This may result in [Invalid_argument] being raised, or in unspecified behaviors."